test/integrationtests/voight_kampff/features/steps/configuration.py

```diff
@@ -20,6 +20,27 @@
 from mycroft.messagebus import Message
 from mycroft.util import resolve_resource_file
 
+CONFIG_CHANGE_TIMEOUT = 10
+
+
+def wait_for_config_change(context, key, expected_value):
+    """Wait for a config change to apply.
+
+    Waits until the specified key has changed into the expected value, if
+    the change delays for too long a TimeoutError will be raised.
+
+    Arguments:
+        context (Context): Behave context of current scenario
+        key (str): key to verify
+        expected_value (Object): The expected value indicating that the change
+                                 has been applied
+    """
+    start_time = time.time()
```

Comment:
It shouldn't cause a problem in this instance but I've been wondering if we should use `time.monotonic()` where ever possible for consistency and to avoid time sync issues?

Are there benefits of using the standard `time.time()`?

Reply:
I considered monotonic but thought the simplicity and readability of time.time() was better than using monotonic but I'll change it.